packages/react-native-branch/android/src/main/java/expo/modules/adapters/branch/BranchPackage.kt

```diff
@@ -0,0 +1,15 @@
+package expo.modules.adapters.branch
+
+import android.content.Context
+import expo.modules.core.BasePackage
+import expo.modules.core.interfaces.ApplicationLifecycleListener
+import expo.modules.core.interfaces.ReactActivityLifecycleListener
+
+class BranchPackage : BasePackage() {
+    override fun createApplicationLifecycleListeners(context: Context?): MutableList<ApplicationLifecycleListener> {
+        return mutableListOf(BranchApplicationLifecycleListener(context))
+    }
```

Comment:
```suggestion
    override fun createApplicationLifecycleListeners(context: Context?): List<ApplicationLifecycleListener> {
        return listOf(BranchApplicationLifecycleListener(context))
    }
```

is it possible to use List here?